frontend/src/hooks/useCardSearch.js

```diff
@@ -0,0 +1,22 @@
+import {useState} from "react";
+import axios from "axios";
+
+export default function useCardSearch(){
+
+    const [cards, setCards] = useState([]);
+
+    const getSpecificCard = (cardName) => {
+        return axios
+            .get('/api/cards?cardName=' + cardName)
+            .then(response => response.data)
+            .then ((data) => setCards(data))
+            .catch((error) => console.log(error))
```

Comment:
you could (later) add error handling in case no card was found

Reply:
thanks, I will consider this for the next pr